test/ShellTest.php

```diff
@@ -261,22 +264,25 @@ public function testReportsErrors($errno, $label)
         \rewind($stream);
         $streamContents = \stream_get_contents($stream);
 
-        $this->assertStringContainsString($label, $streamContents);
-        $this->assertStringContainsString('wheee', $streamContents);
-        $this->assertStringContainsString('line 13', $streamContents);
+        $this->assertSame(<<<EOF
```

Comment:
these were asserting "contains" because older dependencies don't have the exact same output (see 👇)

Reply:
Can you elaborate on this feedback please?

Reply:
Check the "minimum deps" versions of the test suite. Output has change enough over the supported range for our dependencies that the exact string for this output doesn't pass for all of them.

Reply:
Fixed.